hedera-node/cli-clients/src/main/java/com/hedera/services/cli/sign/AccountBalanceSigningUtils.java

```diff
@@ -70,16 +75,16 @@ public static boolean signAccountBalanceFile(
 
             generateSigBalanceFile(signatureFileDestination.toFile(), signature, fileHashByte);
 
-            System.out.println("Generated signature file: " + signatureFileDestination);
+            log.info("Generated signature file: {}", signatureFileDestination);
 
             return true;
         } catch (final SignatureException | InvalidKeyException | IOException e) {
-            System.err.printf(
-                    "signAccountBalanceFile :: Failed to sign file [%s] with exception : [%s]%n", streamFileToSign, e);
+            log.error(
+                    "signAccountBalanceFile :: Failed to sign file [{}] with exception : [{}]%n", streamFileToSign, e);
```

Comment:
`%n` needs to be removed IMO. It encodes a newline for `printf()` but is not needed anymore. Also, I think we should remove the placeholder for the exception.

Same below.